crates/astria-bridge-withdrawer/ethereum/src/AstriaBridgeableERC20.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT or Apache-2.0
```

Comment:
This probably applies to the regular withdrawer as well, but what happens if you don't have enough funds to burn everything you requested?

Reply:
the tx will revert, the tokens are fine. same for native assets